draft-ietf-httpbis-resumable-upload.md

```diff
@@ -180,42 +171,42 @@ Client                                      Server
 
-2) After creation, the following parts are sent using the Upload Appending Procedure ({{upload-appending}}), and the last part of the upload has the `Upload-Complete: ?1` header to indicate the complete transfer.
+2) Afterwards, the following parts are appended ({{upload-appending}}), and the last part of the upload has the `Upload-Complete: ?1` header to indicate the complete transfer.
 
 ~~~
 Client                                      Server
 |                                                |
-| PATCH to upload URL with Upload-Offset         |
-| and Upload-Complete: ?1                        |
+| PATCH to upload resource URL with              |
+| Upload-Offset and Upload-Complete: ?1          |
 |----------------------------------------------->|
 |                                                |
 |                      201 Created on completion |
 |<-----------------------------------------------|
 |                                                |
 ~~~
-{: #fig-upload-appending-procedure-last-chunk title="Upload Appending Procedure Last Chunk"}
+{: #fig-upload-appending-last-chunk title="Upload Append Last Chunk"}
 
-# Upload Creation Procedure {#upload-creation}
+# Upload Creation {#upload-creation}
 
-The Upload Creation Procedure is intended for starting a new upload. A limited form of this procedure MAY be used by the client without the knowledge of server support of the Resumable Uploads Protocol.
+When a resource supports resumable uploads, the first step is creating the upload resource. To be compatible with the widest range of resources, this is accomplished by adding a header field to the request that initiates the upload, `Upload-Complete`.
 
-This procedure is designed to be compatible with a regular upload. Therefore all methods are allowed with the exception of `GET`, `HEAD`, `DELETE`, and `OPTIONS`. All response status codes are allowed. The client is RECOMMENDED to use the `POST` method if not otherwise intended. The server MAY only support a limited number of methods.
+As a consequence, resumable uploads support all HTTP request methods that can carry content, such as `POST`, `PUT`, and `PATCH`. Similarly, the response to the upload request can have any response status code. Both the method(s) and status code(s) supported are determined by the resource.
 
-The request MUST include the `Upload-Complete` header field ({{upload-complete}}). It MUST be set to false if the end of the request body is not the end of the upload. Otherwise, it MUST be set to true. This header field can be used for request identification by a server. The request MUST NOT include the `Upload-Offset` header.
+When present, `Upload-Complete` MUST be set to false if the end of the request content is not the end of the upload. Otherwise, it MUST be set to true. This header field can be used for request identification by a server. The request MUST NOT include the `Upload-Offset` header.
```

Comment:
```suggestion
`Upload-Complete` MUST be set to false if the end of the request content is not the end of the upload. Otherwise, it MUST be set to true. This header field can be used for request identification by a server. The request MUST NOT include the `Upload-Offset` header.
```